ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -30,8 +31,7 @@ import { CVEListQueryResult, cveListQuery } from '../../WorkloadCves/Tables/CVEs
 import { VulnerabilitySeverityLabel } from '../../types';
 import { getWorkloadEntityPagePath } from '../../utils/searchUtils';
 import SeverityCountLabels from '../../components/SeverityCountLabels';
-import CvssFormatted from '../../components/CvssFormatted';
-import DateDistance from '../../components/DateDistance';
+import DateDistance from '../../../../Components/DateDistance';
```

Comment:
One of these things is not like the others. Can this be `'Components/DateDistance'`

Reply:
Ah! I did a global search and replace for the others, must have missed this one. I'll push a fix.